bitnami/kafka/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.19.2
+version: 12.19.3
```

Comment:
Since this is implementing a new parameter, could you increase the `minor` version (i.e. `12.19.3` -> `12.20.0`)?

Reply:
Sure, i just did.